protocol-designer/src/organisms/AnnouncementModal/index.tsx

```diff
@@ -16,7 +16,15 @@ import {
 } from '../../persist'
 import { useAnnouncements } from './announcements'
 
-export const AnnouncementModal = (): JSX.Element => {
+interface AnnouncementModalProp {
+  isViewReleaseNotes?: boolean
+  onClose?: () => void
+}
+
+export const AnnouncementModal = (
+  props: AnnouncementModalProp
```

Comment:
nit: there are multiple props

```suggestion
  props: AnnouncementModalProps
```